# Resources

## Some of my apps
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [SScalculator](https://semiquant.shinyapps.io/SampleSizeCalculator/)  | Comprehensive Sample Size and Power Calculator with range graphs |   |
| [DEAD](https://semiquant.shinyapps.io/DEAD/)  | Diagnostic Estimates and Disease with ROC curves |   |
| [PCRopt](https://semiquant-sq.shinyapps.io/PCRopt/)  | PCR optimization calculator |   |
| [SeqCals](https://semiquant-sq.shinyapps.io/SeeK)  |  Sequencing calculators |   |
| [DrDx.Me](http://drdx.me/)  |  Amplicon data analysis (dev) |   |

## Lab Resources
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [BioZ](https://www.bioz.com/)  | AI Search Engine for Scientific Experimentation |   |
| [BenchSci](https://app.benchsci.com/) | Find Antibodies, AI search engine from all published data and figures |   |
| [Qiita](https://qiita.ucsd.edu/)  | microbial study management platform |   
| [NEB Literature Portal](https://www.neb.com/support/catalog-and-literature-request) | MolBio Litrature | Nice |
| [NEB Molecular Bio Videos](https://www.neb.com/tools-and-resources/video-library) | MolBio vids, e.g. exonucleases activity | Great |
| [NEB Molecular Tools](https://www.neb.com/tools-and-resources) | e.g., PCR fidelity estimator, Double Digest finder | Great |
| [TB mutants](https://www.beiresources.org/BEIMutantSearch.aspx?f_instockflag=In+Stock%23~%23Temporarily+Out+of+Stock) | |

## Papers/reading
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [Researcher App](https://www.researcher-app.com/) | Keep uptodate with papers |   |
| [ResearchRabbit](https://www.researchrabbit.ai/) | Spotify for papers |  |
| [Mendely](https://www.mendeley.com/) | Reference manager |  |
| [Elicit](https://elicit.org/) | AI that finds papers to answer you quetion and summarizes them |  |

## Information
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [bionumbers](https://bionumbers.hms.harvard.edu/search.aspx) | enable you to find in one minute any useful molecular biology number that can be important for your research |   |
| [SciHub](https://sci-hub.tw/) | Get past paywalls for papers  |   |
| [NICD](http://www.nicd.ac.za/) | The national institute for communicable diseases (NICD) Is a resource of knowledge and expertise in regionally relevant communicable diseases to the South African Government, to SADC countries and the African continent. |   |

## Grants
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [Mendeley](https://www.mendeley.com/research-funding/opportunities) | funding opportunities search |   |
| [Postdoc Funding](https://research.jhu.edu/rdt/funding-opportunities/postdoctoral/?utm_source=Nature+Briefing&utm_campaign=63fd5d9973-briefing-dy-20200302&utm_medium=email&utm_term=0_c9dfd39373-63fd5d9973-43998481) | funding opportunities search |   |
| [Junior Researcher](https://research.jhu.edu/rdt/funding-opportunities/postdoctoral/?utm_source=Nature+Briefing&utm_campaign=63fd5d9973-briefing-dy-20200302&utm_medium=email&utm_term=0_c9dfd39373-63fd5d9973-43998481) | funding opportunities search |   |
| [Pivot](https://pivot.proquest.com/) | funding opportunities search  |   |
| [Grantome](http://grantome.com/) | Abstracts from prviously and currently funded federal grants  |   |
| [Dimensions](https://app.dimensions.ai/discover/publication) | Abstracts from prviously and currently funded grants, also pper search and altmetrics  |   |

## Protocols
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [Protocol Exchange](https://protocolexchange.researchsquare.com/) |  |   |
| [Nature Protocols](https://www.nature.com/nprot/) |  |   |
| [CSH Protocols](http://cshprotocols.cshlp.org/) |  |   |
| [Protocols.io](https://www.protocols.io/) |  |   |


## Random
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [SciHub](https://protocolexchange.researchsquare.com/) | get any paper using doi free |   |
| [LibGen](http://www.libgen.is/) | get any book, or magazine free |   |
| [VarNomen](https://varnomen.hgvs.org/recommendations/protein/variant/deletion/) | Variant nomenclature checker |   |
| [DR-TB treatment stats](https://app.powerbi.com/view?r=eyJrIjoiZDhjNDM0YmMtOGExOS00ODIxLWEzMjktZDk0NmI4YTAwODgwIiwidCI6ImY2MTBjMGI3LWJkMjQtNGIzOS04MTBiLTNkYzI4MGFmYjU5MCIsImMiOjh9) |   |  |
| [Shiny Apps](http://shinyapps.org/) | Shocase of shiny apps |  |
| [Google Trends](https://trends.google.com/trends/?geo=US) |  |  |

***

## Bioinformatics/Sequence Data 
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [MycoBrowser](https://mycobrowser.epfl.ch/) | mycobacteria gene browser |   |
| [BioCyc](https://mycobacterium.biocyc.org/MTBH37RV/NEW-IMAGE?type=LOCUS-POSITION&object=NIL&chromosome=CHROM&orgids=MTBH37RV&bp-range=35001/85000) | Gene View of M.tb |   |
| [SRA](http://www.ncbi.nlm.nih.gov/sra) | Archive of sequence data |   |
| [ENA](https://www.ebi.ac.uk/ena) | Archive of sequence data |   |
| [TB Dept](https://depot.tbportals.niaid.nih.gov/#/home) | Comprehensive clinical (including CTs etc), microbiological and genome data from thousands of patients, ability to do analysis on it in the webpage |   |
| [TB portals](https://tbportals.niaid.nih.gov/) | Tuberculosis (TB) data science for public health impact |  |
| [TB and Fever Biomarker Database](https://bm2dx.org/)  | Biomarker Database |   |
| [Lung Cell Atlas](https://asthma.cellgeni.sanger.ac.uk/) | A High-Resolution Reference Map of the Human Lung in Health and Disease |   |
| [Expression Atlas](https://www.ebi.ac.uk/gxa/home) | Gene expression accross species and conditions. Has single cell data as well |   |
| [Lung map](https://www.lungmap.net/)  | un-diseased human (Homo sapiens) and mouse (Mus musculus) tissue |   |
| [AMRFinderPlus](https://www.ncbi.nlm.nih.gov/pathogens/antimicrobial-resistance/AMRFinder/)  | a tool that identifies AMR genes using either protein annotations or nucleotide sequence |   |
| [Patric](https://patricbrc.org/) | Bacterial bioinformatics resource center, loads of pathogens and data, virulence genes, pathwyas, etc. |   |
| [TDSTB](https://gph.niid.go.jp/tgs-tb/) | Total Genotyping Solution for Mycobacterium tuberculosis, and AMR prediction |   |
| [SUSPECT-BDQ](http://biosig.unimelb.edu.au/suspect_bdq/)  | StrUctural Susceptibility PrEdiCTion on BDQ |   |
| [ELIXIR](https://elixir-europe.org/services) | This is basically a resource of resources, like this list but many more. E.g., links to data resources, Biomodels, KEGGAnim, Reactome, STRING, UniProt... | Exxelent |
| [MTB Network Portal](http://networks.systemsbiology.net/mtb/) | A portal for computational modeling program to generate an integrated, predictive gene regulatory network model of host/pathogen interactions. |   |
| [StringDB](https://string-db.org/cgi/input.pl) | STRING is a database of known and predicted protein-protein interactions |   |
| [NIH Human Microbiome Project](https://hmpdacc.org/) |   |   |
| [Academic Torrents](http://academictorrents.com/) | 54TB of research data available |   |
| [TBDB](http://tbdb.bu.edu/tbdb_sysbio/MultiHome.html)  | TB database, old. Has RNAseq data undder different growth conditions, [gene correlation catalouge](http://genome.tbdb.org/tbdb_sysbio/CC/Rv1319c.html) comparitive genomics,  |   |
| [P curve analysis ](http://www.p-curve.com/app4/) | Quick p curve analsysis to evaluate published evidence |  |
| [R Epidemics Consortium](https://www.repidemicsconsortium.org/projects/)  | Some R Epi packages |   |
| [FastGenomics](https://beta.fastgenomics.org/home)  | Sequence data from published papers (scRNA only?) and interactive analysis using Jupyter |   |
| [tbvar](http://genome.igib.res.in/tbvar/index.html) | genome variation resources for Mycobacterium tuberculosis |  |
| [NICID](http://www.nicd.ac.za/) | A national public health institute serving South Africa |  |
| [RECON](http://www.repidemicsconsortium.org/projects/) | The R Epidemics Consortium (RECON) is an international not-for-profit, non-governmental organisation gathering experts in data science, modelling methodology, public health, and software development to create the next generation of analytics tools for informing the response to disease outbreaks, health emergencies and humanitarian crises |  |
| [Qiita](https://qiita.ucsd.edu/) | Qiita (canonically pronounced cheetah) is an entirely open-source microbial study management platform |  |
| [interferome](http://www.interferome.org/interferome/home.jspx) | THE DATABASE OF IFN REGULATED GENES |  |
| [Single Cell Portal](https://singlecell.broadinstitute.org/single_cell) | The Single Cell Portal was developed to facilitate sharing scientific results, and disseminating data generated from single cell technologies. |  |
| [Gene Expression Omnibus](ncbi.nlm.nih.gov/geo/) | GEO is a public functional genomics data repository supporting MIAME-compliant data submissions. |  |
| [archs4](https://maayanlab.cloud/archs4/data.html) | ARCHS4 provides access to gene counts from HiSeq 2000 and HiSeq 2500 platforms for human and mouse experiments. The website allows download of the data in H5 format for programmatic access as well as a 3-dimensional view of the sample space. |  |
| [BioModels](https://www.ebi.ac.uk/biomodels/) | BioModels is a repository of mathematical models of biological and biomedical systems. |  |
| [Reactome](https://reactome.org/) | REACTOME is an open-source, open access, manually curated and peer-reviewed pathway database. |  |
| [DAVID](https://david.ncifcrf.gov/) | The Database for Annotation, Visualization and Integrated Discovery (DAVID) provides a comprehensive set of functional annotation tools for investigators to understand the biological meaning behind large lists of genes. |  |
| [COMBAT-TB NeoDB](https://neodb.sanbi.ac.za/browser/) | fostering Tuberculosis research through integrative analysis using graph database technologies |  |


## Primers/folding
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [UNAfold](http://www.unafold.org) | Great secondary structure prediction |  |
| [RNA](http://rna.informatik.uni-freiburg.de/) | Host of RNA structure tools |  |
| [NUpack](http://www.nupack.org/) | Great secondary structure prediction | |
| [RNAsoft](http://www.rnasoft.ca/cgi-bin/RNAsoft/RNAdesigner/rnadesign.pl) | Design an RNA sequence that folds into a specific secondary structure |  |
| [structureWeb](https://rna.urmc.rochester.edu/RNAstructureWeb/Servers/Predict1/Predict1.html) | Secondary structures |  |
| [primerexplorer](https://primerexplorer.jp/e/) | LAMP primer design |  |
| [primer3](https://www.ncbi.nlm.nih.gov/tools/primer-blast/index.cgi?LINK_LOC=BlastHome) | Primer3 |  |
| [uMelt](https://www.dna-utah.org/umelt/quartz/um.php) | In Sillico Mult Curve |
| [AlphaFold](https://alphafold.ebi.ac.uk/) | Protein folding |  |
| [Freiburg RNA Tools](http://rna.informatik.uni-freiburg.de/) | Freiburg RNA tools provides online access to a series of RNA research tools developed by the Freiburg Bioinformatics Group and colleagues for sequence-structure alignments (LocARNA, CARNA, MARNA), clustering (ExpaRNA), interaction prediction (IntaRNA, CopraRNA, metaMIR), identification of homologs (GLASSgo), sequence design (AntaRNA, INFORNA, SECISDesign), CRISPR repeat analyses (CRISPRmap), and many more tasks |  |
| [kinefold](http://kinefold.curie.fr/) | RNA transcription and foling animated |  |


<!-- 

| []() | | --- |
## UMI/ONT data handling
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [gencore](https://github.com/OpenGene/gencore) | generating consensus reads for paired-end next-generation sequencing with/without UMIs |   |
| [mageri](https://mageri.readthedocs.io/en/latest/body.html#) | MAGERI is an all-in-one software for analysis of targeted genome re-sequencing data for libraries prepared with novel unique molecular identifier tagging technology..extracts UMI sequences, performes primer and adapter matching and trimming, assembles molecular consensuses, alignes them to reference sequences and calls variants |   |
| [UMICollapse](https://github.com/Daniel-Liu-c0deb0t/UMICollapse) | Accelerating the deduplication and collapsing process for reads with Unique Molecular Identifiers (UMI) |   |
| [fgbio](http://fulcrumgenomics.github.io/fgbio/tools/latest/) | Tools for working with genomic and high throughput sequencing data. Does UMI consensus but cant handel indels |   |
| [medaka](https://github.com/nanoporetech/medaka) | create a consensus sequence from nanopore sequencing data |   |
| [racon](https://github.com/isovic/racon) | Racon is intended as a standalone consensus module to correct raw contigs generated by rapid assembly methods which do not include a consensus step |   |
| [Longread_umi](https://github.com/SorenKarst/longread_umi) | Enabling high-accuracy long-read amplicon sequences using unique molecular identifiers with Nanopore or PacBio sequencing. – A collection of scripts for processing longread UMI data. |   |
| [UMI-Reducer](xx) | UMI-Reducer is a method for processing and differentiating PCR duplicates from biological duplicates and bias-free estimation of mRNA abundance in the sample. One UMI, |   |
| [Alfred](https://www.gear-genomics.com/docs/alfred/cli/#bam-consensus-computation) | BAM Statistics, Feature Counting and Feature Annotation, BAM Consensus Computation, Haplotype-specific BAM files |   |
| [rageseq](https://github.com/KCCG/rageseq) | https://www.biorxiv.org/content/early/2018/09/24/424945 |   |
| [minmap2](https://github.com/lh3/minimap2#map-long-genomic) | versatile sequence alignment program that aligns DNA or mRNA sequences against a large reference database |   |
| [calib](https://github.com/vpc-ccg/calib) | Calib clusters barcode tagged paired-end reads based on their barcode and sequence similarity. |   |
| [dunovo](https://github.com/galaxyproject/dunovo) | processing duplex sequencing data without the use of a reference genome |   |
| [produse](https://produse.readthedocs.io/en/latest/index.html) | ProDuSe is a variant caller designed for use with libraries prepared and sequenced using semi-degenerate barcoded adapters. These barcodes allow PCR and optical duplicates which were derived from the same starting molecule to be flagged and merged into a single consensus sequence. |   |
| [iDES](https://cappseq.stanford.edu/ides/download.php#usage) | method for the suppression of stereotypical background artifacts in high-throughput sequencing data.  |   |
| [Connor](https://github.com/umich-brcf-bioinf/Connor) | A command-line tool to deduplicate bam files based on custom, inline barcoding. |   |
| [Duplex-Sequencing](https://github.com/KennedyLabUW/Duplex-Sequencing) | xxx |   |
| [FERMI](https://github.com/liggettla/FERMI/wiki) | FERMI contains set of tools to analyze unique molecular identifier (UMI) tagged, amplicon captured, genomic DNA sequence data. |   |
| [fastqp](https://github.com/OpenGene/fastp#unique-molecular-identifier-umi-processing) | A tool designed to provide fast all-in-one preprocessing for FastQ files.  |  good |
| [SolidBin](https://github.com/sufforest/SolidBin) | Improving Metagenome Binning with Semi-supervised Normalized Cut  |   |
| [DADA2](https://benjjneb.github.io/dada2/index.html) | Fast and accurate sample inference from amplicon data with single-nucleotide resolution  |   |
| [sicelore](https://github.com/ucagenomix/sicelore#bc-assignment) | a suite of tools dedicated to the cell barcode and UMI (unique molecular identifier) assignment and bioinformatics analysis of highly multiplexed single cell Nanopore long read sequencing data  |   |
| [presto](https://presto.readthedocs.io/en/stable/examples/workflows.html) | pRESTO performs all stages of raw sequence processing prior to alignment against reference germline sequences.  |   |
| [umi-varcal](https://gitlab.com/vincent-sater/umi-varcal-master) | UMI-based variant caller that efficiently improves low-frequency variant detection in paired-end sequencing NGS libraries  |   |
| [zUMIs](https://github.com/sdparekh/zUMIs) | zUMIs is a fast and flexible pipeline to process RNA-seq data with (or without) UMIs.  |   |
 -->










