# Resources

## Lab Resources
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [BioZ](https://www.bioz.com/)  | AI Search Engine for Scientific Experimentation |   |
| [BenchSci](https://app.benchsci.com/) | Find Antibodies, AI search engine from all published data and figures |   |
| [Qiita](https://qiita.ucsd.edu/)  | microbial study management platform |   
| [NEB Literature Portal](https://www.neb.com/support/catalog-and-literature-request) | MolBio Litrature | Nice |
| [NEB Molecular Bio Videos](https://www.neb.com/tools-and-resources/video-library) | MolBio vids, e.g. exonucleases activity | Great |
| [NEB Molecular Tools](https://www.neb.com/tools-and-resources) | e.g., PCR fidelity estimator, Double Digest finder | Great |
| [TB mutants](https://www.beiresources.org/BEIMutantSearch.aspx?f_instockflag=In+Stock%23~%23Temporarily+Out+of+Stock) | |
| Other | |
| [Grantome](www.grantome.com) |   |   |
| [NICD](http://www.nicd.ac.za/) | The national institute for communicable diseases (NICD) Is a resource of knowledge and expertise in regionally relevant communicable diseases to the South African Government, to SADC countries and the African continent. |   |
| [bionumberis](https://bionumbers.hms.harvard.edu/search.aspx) | enable you to find in one minute any useful molecular biology number that can be important for your research |   |


## Grants
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [Mendeley](https://www.mendeley.com/research-funding/opportunities) | funding opportunities search |   |
| [Postdoc Funding](https://research.jhu.edu/rdt/funding-opportunities/postdoctoral/?utm_source=Nature+Briefing&utm_campaign=63fd5d9973-briefing-dy-20200302&utm_medium=email&utm_term=0_c9dfd39373-63fd5d9973-43998481) | funding opportunities search |   |
| [Junior Researcher](https://research.jhu.edu/rdt/funding-opportunities/postdoctoral/?utm_source=Nature+Briefing&utm_campaign=63fd5d9973-briefing-dy-20200302&utm_medium=email&utm_term=0_c9dfd39373-63fd5d9973-43998481) | funding opportunities search |   |
| [Pivot](https://pivot.proquest.com/) | funding opportunities search  |   |
| [Grantome](http://grantome.com/) | Abstracts from prviously and currently funded federal grants  |   |


## Bioinformatics/Sequence Data 
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [MycoBrowser](https://mycobrowser.epfl.ch/) | mycobacteria gene browser |   |
| [BioCyc](https://mycobacterium.biocyc.org/MTBH37RV/NEW-IMAGE?type=LOCUS-POSITION&object=NIL&chromosome=CHROM&orgids=MTBH37RV&bp-range=35001/85000) | Gene View of M.tb |   |
| [SRA](http://www.ncbi.nlm.nih.gov/sra) | Archive of sequence data |   |
| [ENA](https://www.ebi.ac.uk/ena) | Archive of sequence data |   |
| [TB Dept](https://depot.tbportals.niaid.nih.gov/#/home) | Comprehensive clinical (including CTs etc), microbiological and genome data from thousands of patients, ability to do analysis on it in the webpage |   |
| [TB and Fever Biomarker Database](https://bm2dx.org/)  | Biomarker Database |   |
| [Lung Cell Atlas](https://asthma.cellgeni.sanger.ac.uk/) | A High-Resolution Reference Map of the Human Lung in Health and Disease |   |
| [Expression Atlas](https://www.ebi.ac.uk/gxa/home) | Gene expression accross species and conditions. Has single cell data as well |   |
| [Lung map](https://www.lungmap.net/)  | un-diseased human (Homo sapiens) and mouse (Mus musculus) tissue |   |
| [AMRFinderPlus](https://www.ncbi.nlm.nih.gov/pathogens/antimicrobial-resistance/AMRFinder/)  | a tool that identifies AMR genes using either protein annotations or nucleotide sequence |   |
| [Patric](https://patricbrc.org/) | Bacterial bioinformatics resource center, loads of pathogens and data, virulence genes, pathwyas, etc. |   |
| [TDSTB](https://gph.niid.go.jp/tgs-tb/) | Total Genotyping Solution for Mycobacterium tuberculosis, and AMR prediction |   |
| [SUSPECT-BDQ](http://biosig.unimelb.edu.au/suspect_bdq/)  | StrUctural Susceptibility PrEdiCTion on BDQ |   |
| [ELIXIR](https://elixir-europe.org/services) | This is basically a resource of resources, like this list but many more. E.g., links to data resources, Biomodels, KEGGAnim, Reactome, STRING, UniProt... | Exxelent |
| [MTB Network Portal](http://networks.systemsbiology.net/mtb/) | A portal for computational modeling program to generate an integrated, predictive gene regulatory network model of host/pathogen interactions. |   |
| [StringDB](https://string-db.org/cgi/input.pl) | STRING is a database of known and predicted protein-protein interactions |   |
| [NIH Human Microbiome Project](https://hmpdacc.org/) |   |   |
| [Academic Torrents](http://academictorrents.com/) | 54TB of research data available |   |
| [TBDB](http://tbdb.bu.edu/tbdb_sysbio/MultiHome.html)  | TB database, old. Has RNAseq data undder different growth conditions, [gene correlation catalouge](http://genome.tbdb.org/tbdb_sysbio/CC/Rv1319c.html) comparitive genomics,  |   |
| [P curve analysis ](http://www.p-curve.com/app4/) | Quick p curve analsysis to evaluate published evidence |  |
| [R Epidemics Consortium](https://www.repidemicsconsortium.org/projects/)  | Some R Epi packages |   |

## UMI/ONT data handling
| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [gencore](https://github.com/OpenGene/gencore) | generating consensus reads for paired-end next-generation sequencing with/without UMIs |   |
| [mageri](https://mageri.readthedocs.io/en/latest/body.html#) | MAGERI is an all-in-one software for analysis of targeted genome re-sequencing data for libraries prepared with novel unique molecular identifier tagging technology..extracts UMI sequences, performes primer and adapter matching and trimming, assembles molecular consensuses, alignes them to reference sequences and calls variants |   |
| [UMICollapse](https://github.com/Daniel-Liu-c0deb0t/UMICollapse) | Accelerating the deduplication and collapsing process for reads with Unique Molecular Identifiers (UMI) |   |
| [fgbio](http://fulcrumgenomics.github.io/fgbio/tools/latest/) | Tools for working with genomic and high throughput sequencing data. Does UMI consensus but cant handel indels |   |
| [medaka](https://github.com/nanoporetech/medaka) | create a consensus sequence from nanopore sequencing data |   |
| [racon](https://github.com/isovic/racon) | Racon is intended as a standalone consensus module to correct raw contigs generated by rapid assembly methods which do not include a consensus step |   |
| [Longread_umi](https://github.com/SorenKarst/longread_umi) | Enabling high-accuracy long-read amplicon sequences using unique molecular identifiers with Nanopore or PacBio sequencing. â€“ A collection of scripts for processing longread UMI data. |   |
| [UMI-Reducer](xx) | UMI-Reducer is a method for processing and differentiating PCR duplicates from biological duplicates and bias-free estimation of mRNA abundance in the sample. One UMI, |   |
| [Alfred](https://www.gear-genomics.com/docs/alfred/cli/#bam-consensus-computation) | BAM Statistics, Feature Counting and Feature Annotation, BAM Consensus Computation, Haplotype-specific BAM files |   |
| [rageseq](https://github.com/KCCG/rageseq) | https://www.biorxiv.org/content/early/2018/09/24/424945 |   |
| [minmap2](https://github.com/lh3/minimap2#map-long-genomic) | versatile sequence alignment program that aligns DNA or mRNA sequences against a large reference database |   |
| [calib](https://github.com/vpc-ccg/calib) | Calib clusters barcode tagged paired-end reads based on their barcode and sequence similarity. |   |
| [dunovo](https://github.com/galaxyproject/dunovo) | processing duplex sequencing data without the use of a reference genome |   |
| [produse](https://produse.readthedocs.io/en/latest/index.html) | ProDuSe is a variant caller designed for use with libraries prepared and sequenced using semi-degenerate barcoded adapters. These barcodes allow PCR and optical duplicates which were derived from the same starting molecule to be flagged and merged into a single consensus sequence. |   |
| [iDES](https://cappseq.stanford.edu/ides/download.php#usage) | method for the suppression of stereotypical background artifacts in high-throughput sequencing data.  |   |
| [Connor](https://github.com/umich-brcf-bioinf/Connor) | A command-line tool to deduplicate bam files based on custom, inline barcoding. |   |
| [Duplex-Sequencing](https://github.com/KennedyLabUW/Duplex-Sequencing) | xxx |   |
| [FERMI](https://github.com/liggettla/FERMI/wiki) | FERMI contains set of tools to analyze unique molecular identifier (UMI) tagged, amplicon captured, genomic DNA sequence data. |   |
| [fastqp](https://github.com/OpenGene/fastp#unique-molecular-identifier-umi-processing) | A tool designed to provide fast all-in-one preprocessing for FastQ files.  |  good |
| [SolidBin](https://github.com/sufforest/SolidBin) | Improving Metagenome Binning with Semi-supervised Normalized Cut  |   |
| [DADA2](https://benjjneb.github.io/dada2/index.html) | Fast and accurate sample inference from amplicon data with single-nucleotide resolution  |   |
| [sicelore](https://github.com/ucagenomix/sicelore#bc-assignment) | a suite of tools dedicated to the cell barcode and UMI (unique molecular identifier) assignment and bioinformatics analysis of highly multiplexed single cell Nanopore long read sequencing data  |   |
| [presto](https://presto.readthedocs.io/en/stable/examples/workflows.html) | pRESTO performs all stages of raw sequence processing prior to alignment against reference germline sequences.  |   |
| [umi-varcal](https://gitlab.com/vincent-sater/umi-varcal-master) | UMI-based variant caller that efficiently improves low-frequency variant detection in paired-end sequencing NGS libraries  |   |
| [zUMIs](https://github.com/sdparekh/zUMIs) | zUMIs is a fast and flexible pipeline to process RNA-seq data with (or without) UMIs.  |   |


***


| Name(link) | Use/Info | Note |
| --- | --- | --- |
| [xx](xx) | xxx  |   |
| [xx](xx) | xxx  |   |
| [xx](xx) | xxx  |   |
| [xx](xx) | xxx  |   |
| [xx](xx) | xxx  |   |
| [xx](xx) | xxx  |   |





